File: segundo-semestre/lab121/tarea1/c_sharp/interes_simple.cs
class MainClass {
  public static void Main(string[] args) {
    double interes;
    double capital;
    double tiempo;
    double pin;

    System.Console.WriteLine("Ingrese interes: ");
    string interes_string = System.Console.ReadLine();
    interes = float.Parse(interes_string);

    System.Console.WriteLine("Ingrese capital: ");
    string capital_string = System.Console.ReadLine();
    capital = float.Parse(capital_string);

    System.Console.WriteLine("Ingrese tiempo en años: ");
    string tiempo_string = System.Console.ReadLine();
    tiempo = float.Parse(tiempo_string);

    pin = (interes * capital) / tiempo;

    System.Console.WriteLine("El interes simple es de: " + pin);
  }
}

File: segundo-semestre/lab121/tarea1/java/interes_simple.java
import java.util.Scanner;

class Interes_simple {
  public static void main(String args[]) {
    Scanner reader = new Scanner(System.in);

    double interes;
    double capital;
    double tiempo;
    double pin;

    System.out.println("Ingrese interes: ");
    interes = reader.nextDouble();

    System.out.println("Ingrese capital: ");
    capital = reader.nextDouble();

    System.out.println("Ingrese tiempo en años: ");
    tiempo = reader.nextDouble();

    pin = ((interes * capital) / tiempo);
    System.out.println("Su interes simple es de: " + pin);
  }
}
